runner/upstream/entrypoint.sh

```diff
@@ -20,6 +20,20 @@ function call() {
     curl --retry 5 --retry-delay 5 --retry-connrefused --fail -s -X POST -d "${PAYLOAD}" -H 'Accept: application/json' -H "Authorization: Bearer ${BEARER_TOKEN}" "${CALLBACK_URL}" || echo "failed to call home: exit code ($?)"
 }
 
+function systemInfo() {
+    if [ -f "/etc/os-release" ];then
+        . /etc/os-release
+    fi
+    OS_NAME=${NAME:-""}
+    OS_VERSION=${VERSION_ID:-""}
```

Comment:
Just thinking about if we could set some meaningful default values for `$NAME` and `$VERSION_ID` maybe during build as `ENV` in the Dockerfile. Then a user could still overwrite these in their podTemplateSpec in the provdider config if needed.

Reply:
All of this stuff is strictly informational. It's nice to have, but not strictly necessary in terms of functionality. Simply put, it has no baring on functionality, but if a user does a:

```bash
garm-cli runner show <runner ID>
``` 

that command will show the real OS and version, not something we infer from the image itself.

This was always the way I hoped this information would be sent upstream (runners would send that info as part of their installation process), but only now got around to doing it.

The idea is that by default, it should be fine to have an empty string for this stuff. But if we can safely determine the OS and version , we should send that to garm.

The `/etc/os-release` file has been available for a very long time. So I expect it to be present in any container image that is based of any OS that;s not `scratch`.